src/routines/routines.js

```diff
@@ -0,0 +1,3 @@
+import { createRoutine } from 'redux-saga-routines';
```

Comment:
I don't particularly like this naming. it imports as `import { fetchSystemIntakes } from 'routines/routines'. It can be confusing, but I don't have any good suggestions off the top of my head. I will continue thinking.

Reply:
since we will only have one routine for each endpoint (presumably), we probably don't need a routines package. We could either put all the routines in one file or put each one in the associated saga file. I'll noodle on it.